src/main/resources/css/common.css

```diff
@@ -61,14 +61,26 @@ span {
     height:40px;
     width: 40px;
 }
+.c-navbar {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px #111A3133 solid;
+    width: 100%;
+    height:56px;
+}
+.c-navbar > div {
+    width: 100%;
+    height: 100%;
+}
 .navbar {
     display: flex;
     align-items: center;
     justify-content: space-evenly;
     width: 100%;
-    height: 56px;
+    max-width: 900px;
+    height: 100%;
```

Comment:
I think this one causes the nav bar to display incorrectly on mobile version. When I remove height: 100%, it looks like this (with the offset on top)  which can be rendered accordingly, too
![Capture](https://user-images.githubusercontent.com/34975907/89212247-24a52b80-d591-11ea-83a7-5b206929e46c.PNG)

Reply:
I forgot to push up that change. look at the latest commit, it should be working